conans/test/utils/tools.py

```diff
@@ -636,13 +651,9 @@ def chdir(self, newdir):
         finally:
             self.current_folder = old_dir
 
-    def get_conan_api_v2(self, user_io=None):
-        if user_io:
-            self.out = user_io.out
-        else:
-            self.out = BufferConanOutput()
-        user_io = user_io or MockedUserIO(self.users, out=self.out)
-        conan = ConanAPIV2(cache_folder=self.cache_folder, output=self.out, user_io=user_io,
+    def get_conan_api_v2(self):
+        user_io = MockedUserIO(self.users, out=self.out)
```

Comment:
If it is going to be redirected, why necessary to pass self.out? I don't fully get it.